packages/location-management/src/components/LocationTagView/tests/index.test.tsx

```diff
@@ -0,0 +1,70 @@
+import { Provider } from 'react-redux';
+import { store } from '@opensrp/store';
+import { mount, shallow } from 'enzyme';
+import { history } from '@onaio/connected-reducer-registry';
+import { Router } from 'react-router';
+import React from 'react';
+import LocationTagView from '..';
+import { act } from 'react-dom/test-utils';
+import flushPromises from 'flush-promises';
+import fetch from 'jest-fetch-mock';
+import { sampleLocationTagPayload } from '../../LocationTagAddEdit/tests/fixtures';
+import { notification } from 'antd';
+
+describe('containers/pages/locations/LocationTagView', () => {
+  beforeEach(() => {
+    fetch.resetMocks();
+  });
+  it('renders without crashing', async () => {
+    shallow(
+      <Router history={history}>
+        <LocationTagView />
+      </Router>
+    );
+  });
+
+  it('works correctly with store', async () => {
+    fetch.mockResponse(JSON.stringify([sampleLocationTagPayload]));
+    const wrapper = mount(
+      <Provider store={store}>
+        <Router history={history}>
+          <LocationTagView />
+        </Router>
+      </Provider>
+    );
+
+    await act(async () => {
+      await flushPromises();
+      wrapper.update();
+    });
+    // test search input works
+    const input = wrapper.find('input').first();
+    input.simulate('change', { target: { value: 'Sample' } });
+    await act(async () => {
+      wrapper.update();
+    });
+    expect(input.instance().value).toEqual('Sample');
+    wrapper.unmount();
```

Comment:
I might have missed sth, but how does this confirm that the component works correctly with the store?

Reply:
@KipSigei  any idea on this  ?

Reply:
@ashfahan you should be able to resolve this, by asserting that the data mocked in the fetch response, is rendered on the page once the component mounts.

Reply:
@p-netm Since we've switched to using `useDispatch` and not the connect HOC as in the previous pattern we are used to, I don't see how the test case `works correctly with store` is now relevant. I think it should just be `renders correctly`